semgrep/semgrep/target_manager.py

```diff
@@ -113,36 +128,26 @@ def _expand_dir(
                         encoding="utf-8",
                         stderr=subprocess.DEVNULL,
                     )
-                except subprocess.CalledProcessError:
-                    raise NotGitProjectError(
-                        f"{curr_dir.resolve()} is not a git repository."
-                    )
 
-                tracked = TargetManager._parse_output(tracked_output, curr_dir)
-                untracked_unignored = TargetManager._parse_output(
-                    untracked_output, curr_dir
-                )
+                    tracked = _parse_output(tracked_output, curr_dir)
+                    untracked_unignored = _parse_output(untracked_output, curr_dir)
 
-                expanded = expanded.union(tracked)
-                expanded = expanded.union(untracked_unignored)
+                    expanded = expanded.union(tracked)
+                    expanded = expanded.union(untracked_unignored)
+                except subprocess.CalledProcessError:
```

Comment:
I think we'll need to catch a separate exception if `git` doesn't exist on their system:

```
In [45]: try: 
    ...:     subprocess.check_output('ladida') 
    ...: except subprocess.CalledProcessError: 
    ...:     print("CAUGHT") 
    ...:

...

FileNotFoundError: [Errno 2] No such file or directory: 'ladida': 'ladida'
```